### ISSUE TITLE
30.05.  24 oclock still auditing

### ISSUE THREAD

**Issue by monteray99** [hpool-dev/chia-miner#528]
hy ,

just want to know wich timezone is meant by 30.05. / 24 o´clock for automatic pass the auditing.

Thanks

**Comment by masjlab00m199** [comment]
their side has been 24 hours 34 / 5

**Comment by turgutcaa** [comment]
> hy ,
> 
> just want to know wich timezone is meant by 30.05. / 24 o´clock for automatic pass the auditing.
> 
> Thanks

no nothing has changed

**Comment by Eioz76** [comment]
same here still auditing

**Comment by krsticsmederevac** [comment]
No change, still the same, auditing...

**Comment by rakesh2906** [comment]
Same here, It's still auditing.

**Comment by ndsutter** [comment]
Nothing changed, still auditing.  Can we get an update from the devs on when they expect to resolve?

**Comment by rollerbones** [comment]
Better communication would be nice.  Even quick updates somewhere just to know someone is really working on it.

**Comment by Exzellenz** [comment]
Yes any update would be highly appreciated i Currently let a small amount of plots run for hpool and the rest solo, but i do not know if it makes sense to let anything run on hpool at all in auditing state.

**Comment by fen1x-fen1x** [comment]
Somebody try to rebind Signature?

**Comment by piszpunta** [comment]
"The signature data is only valid for 24 hours. Please use the tool to obtain the signature data again if the binding timed out.
If you have any questions, please contact the official customer service or the official email: service@hpool.cn."

should i do unbinding and binding again?

**Comment by turgutcaa** [comment]
> "The signature data is only valid for 24 hours. Please use the tool to obtain the signature data again if the binding timed out.
> If you have any questions, please contact the official customer service or the official email: [service@hpool.cn](mailto:service@hpool.cn)."
> 
> should i do unbinding and binding again?

I don't think there will be any change.

**Comment by szamaq** [comment]
I just put it here... https://www.youtube.com/watch?v=pK8Rs54h2l4

Anyway, I give hpool time one - two more days and if I nothing changes there is no reason to stay in hpool but going solo.

**Comment by rakesh2906** [comment]
Quick update: My audit is successful now. It took me 1.5 days.

**Comment by paulshark30** [comment]
аудит успешно завершен, но участки до сих пор без подписи ((

**Comment by turgutcaa** [comment]
Yes audit approved but plots appear unsigned.

**Comment by xicexice** [comment]
垃圾黑心矿池偷我们算力
